docs/quickstarts/nextjs.mdx

```diff
@@ -237,8 +237,6 @@ To see an example application that demonstrates user authentication, and session
 
     <Cards title="Next.js SDK Reference" description="Learn more about additional Next.js methods." link="/docs/references/nextjs/overview" cta="Learn More" />
 
-    <Cards title="Use Clerk with Next.js 12 and older" description="Learn how to use Clerk with older versions of Next.js." link="/docs/references/nextjs/usage-with-older-versions" cta="Learn More" />

```

Comment:
we had discussed the removal of this in the previous nextjs quickstart update - basically, I don't believe its a step in a user's journey and therefore, doesn't need to be included in the nextjs quickstart.